pkg/scheduler/framework/runtime/framework_test.go

```diff
@@ -72,7 +72,7 @@ var cmpOpts = []cmp.Option{
 		if s1 == nil || s2 == nil {
 			return s1.IsSuccess() && s2.IsSuccess()
 		}
-		return s1.Code() == s2.Code() && s1.Plugin() == s2.Plugin() && s1.Message() == s2.Message()
+		return s1.Code() == s2.Code() && cmp.Equal(s1.Plugin(), s2.Plugin()) && s1.Message() == s2.Message()
```

Comment:
Maybe out of this pr's scope, but why can't we use `s1.Equal(s2)` here?

Reply:
the older s1.Plugin() is string, this pr defined as slice, so need modify ut

Reply:
Yes, but since we already have `Equal` function to compare two status, why would we have another `cmpOpts` for comparing? I'm not clear of the context here.

Reply:
when i use `s1.Equal(s2)`, some ut failed, i try figure out

Reply:
It's ok, when using `s1.Equal(s2)`, we'll lose some information in `diff`, so let's just leave it here.